src/wolfio.c

```diff
@@ -694,7 +697,24 @@ int wolfIO_Recv(SOCKET_T sd, char *buf, int sz, int rdFlags)
 {
     int recvd;
 
+#ifdef FUSION_RTOS
+    int err;
+retry:
+    recvd = (int)RECV_FUNCTION(sd, buf, sz, rdFlags, &err);
+
+    if(recvd < 0) {
+       if(err==63)
```

Comment:
Can we avoid a hard coded `63`? Is there an error code lookup available or even just add one for this case? Please cleanup / resolve the TODO below at 714.

Reply:
Removed (customer ultimately used a custom I/O callback these changes were not necessary in the porting commit.